Sources/TokamakDOM/Styles/ListStyle.swift

```diff
@@ -0,0 +1,24 @@
+// Copyright 2020 Tokamak contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//  Created by Jed Fox on 06/30/2020.
+//
+
+import TokamakCore
+
+public typealias DefaultListStyle = TokamakCore.DefaultListStyle
+public typealias PlainListStyle = TokamakCore.PlainListStyle
+public typealias InsetListStyle = TokamakCore.InsetListStyle
+public typealias GroupedListStyle = TokamakCore.GroupedListStyle
+public typealias InsetGroupedListStyle = TokamakCore.InsetGroupedListStyle
```

Comment:
These re-exports should be moved to `Core.swift` probably. I'm also getting `invalid redeclaration` build errors, which seem to be reproducible on CI.